admin_manual/ai/app_text2image_stablediffusion2.rst

```diff
@@ -0,0 +1,60 @@
+=========================================================
+App: Local Image Generation (text2image_stablediffusion2)
+=========================================================
+
+.. _ai-app-text2image_stablediffusion2:
+
+The *text2image_stablediffusion2* app is one of the apps that provide image generation functionality in Nextcloud and act as an image generation backend for the :ref:`Nextcloud Assistant app<ai-app-assistant>` and :ref:`other apps making use of the image generation functionality. The *text2image_stablediffusion2* app specifically runs only open source models and does so entirely on-premises. Nextcloud can provide customer support upon request, please talk to your account manager for the possibilities.
+
+Requirements
+------------
+
+* This app is built as an External App and thus depends on AppAPI v3.1.0 or higher
+* Nextcloud AIO is supported
+* We currently support NVIDIA GPUs and x86_64 CPUs
+* CUDA >= v12.2 on your host system
+* GPU Sizing
+
+   * A NVIDIA GPU with at least 8GB VRAM
+
+* CPU Sizing
+
+   * At least 8GB of system RAM
+   * The more cores you have and the more powerful the CPU the better, we recommend 10-20 cores
+   * The app will hog all cores by default, so it is usually better to run it on a separate machine
+
+Installation
+------------
+
+0. Make sure the :ref:`Nextcloud Assistant app<ai-app-assistant>` is installed
+1. :ref:`Install AppAPI and setup a Deploy Demon<ai-app_api>`
+2. Install the "Local large language model" ExApp via the "Apps" page in the Nextcloud web admin user interface
+
+
+Scaling
+-------
+
+It is currently not possible to scale this app, we are working on this. Based on our calculations an instance has a rough capacity of 120 image requests per hour (each user request can be for multiple images). However, this number is based on theory and we do appreciate real-world feedback on this.
```

Comment:
the testing setup could be mentioned here for a rough idea

Reply:
Can you elaborate?

Reply:
I mean some details of the machine on which 120 images/h was achieved, the gpu and cpu perhaps.